llvm/docs/Extensions.rst

```diff
@@ -451,6 +451,76 @@ Example:
    .uleb128  .LBB_END0_1-.LBB0_1          # BB_1 size
    .byte     y                            # BB_1 metadata
 
+``SHT_LLVM_PGO_BB_ADDR_MAP`` Section (pgo basic block address map)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This section has all the same functionality as ``SHT_LLVM_BB_ADDR_MAP`` with
+additional data relating to PGO analyses. Supported analyses that can be
+emitted currently are Function Entry Count, Basic Block Frequencies, and Branch
+Probabilities.
+
+Each analysis is enabled or disabled via a bit in the feature byte. Currently
+those bits are:
+* 0 - Function Entry Count
+* 1 - Basic Block Frequencies
+* 2 - Branch Probabilities
+
```

Comment:
The meaning of "block frequencies" and "branch probabilities" can be pretty unintuitive for people not familiar with LLVM internals. So probably a good idea to add a sentence and/or some references explaining those values. "Block Frequencies are integer numbers expressing the relative execution count compared tot the the frequency of the first block of the function. See also ..."? "Branch probabilities value express the numerator to an assumed fraction with a fixed denominator being 1<<31. See also BranchProbability.h"?

Reply:
Added explanations to each of these and gave the path for headers on BlockFrequency and BranchProbabilities